v2/cmd/nuclei/main.go

```diff
@@ -90,6 +90,13 @@ on extensive configurability, massive extensibility and ease of use.`)
 
 		flagSet.StringVarP(&options.DiskExportDirectory, "markdown-export", "me", "", "directory to export results in markdown format"),
 		flagSet.StringVarP(&options.SarifExport, "sarif-export", "se", "", "file to export results in SARIF format"),
+
+		flagSet.StringVar(&options.ESExport, "es-ip", "", "the ip address for elasticsearch . Ex: 127.0.0.1"),
+		flagSet.IntVar(&options.ESPort, "es-port", 9200, "the port number on which elasticsearch is running, by default 9200"),
```

Comment:
Normally the default values provided ahould automatically shown when nuclei is invoked with `-h`. Could you please make sure that it does and if so remove the "defaults" from the description?

Reply:
I checked, the default is provided by `-h` option. I removed "defaults" from description and pushed it.